examples/js/effects/VREffect.js

```diff
@@ -19,7 +19,7 @@ THREE.VREffect = function ( renderer, onError ) {
 	var renderRectL, renderRectR;
 	var eyeFOVL, eyeFOVR;
 
-	function gotVRDisplays( displays ) {
```

Comment:
I'm thinking to add this to the wiki.

https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style%E2%84%A2

Looks good to you?

## Functions

- The function keyword should always be followed with a space.
- The function name should be followed with an opening parenthesis.

good:

    function ( arg1, arg2 ) {

    }

    function functionName( arg1, arg2 ) {

    }

bad:

    function( arg1, arg2 ) {

    }

    function functionName ( arg1, arg2 ) {

    }